src/python/pants/notes/master.rst

```diff
@@ -4,6 +4,131 @@ Master Pre-Releases
 This document describes development releases which occur weekly from master, and which have
 not undergone the vetting associated with ``stable`` releases.
 
+1.25.0.dev0 (01/03/2020)
+------------------------
+
+API Changes
+~~~~~~~~~~~
+
+* Deprecate `--fmt-skip` and `--lint-skip` (#8900)
+  `Issue #8346 <https://github.com/pantsbuild/pants/issues/8346>`_
+  `PR #8900 <https://github.com/pantsbuild/pants/pull/8900>`_
+
+* Add `--skip` option to every linter/formatter subsystem (#8898)
+  `PR #8898 <https://github.com/pantsbuild/pants/pull/8898>`_
+
+* Add ESLint subsystem and deprecate `node-distribution` ESLint options (#8891)
+  `PR #8891 <https://github.com/pantsbuild/pants/pull/8891>`_
+
+* Add `--black-skip`, `--isort-skip`, and `--flake8-skip` (#8888)
+  `PR #8888 <https://github.com/pantsbuild/pants/pull/8888>`_
+
+* Deprecate running `python-eval` by default (#8889)
+  `PR #8889 <https://github.com/pantsbuild/pants/pull/8889>`_
+
+* Add `scrooge-linter` subsystem and deprecate `lint-thrift` options (#8894)
+  `PR #8894 <https://github.com/pantsbuild/pants/pull/8894>`_
+
+* Add `checkstyle` subsystem and deprecate `--lint-checkstyle-configuration` (#8896)
+  `PR #8896 <https://github.com/pantsbuild/pants/pull/8896>`_
+
+* Deprecate `option.custom_types.dict_option` and `list_option` (#8885)
+  `PR #8885 <https://github.com/pantsbuild/pants/pull/8885>`_
+
+* Add `scalafix`, `scalafmt`, and `scalastyle` subsystems and deprecate task-level config options (#8886)
+  `PR #8886 <https://github.com/pantsbuild/pants/pull/8886>`_
+
+* Stop using Java 6 and 7 internally (#8657)
+  `PR #8657 <https://github.com/pantsbuild/pants/pull/8657>`_
+
+* Bump default MyPy from 0.740 to 0.761 (#8866)
+  `PR #8866 <https://github.com/pantsbuild/pants/pull/8866>`_
+
+Bugfixes
+~~~~~~~~
+
+* Use absolute path when running interactive process in tempdir (#8895)
+  `PR #8895 <https://github.com/pantsbuild/pants/pull/8895>`_
+
+* Fix `./pants lint2` for Black and isort (#8877)
+  `PR #8877 <https://github.com/pantsbuild/pants/pull/8877>`_
+  `PR #8823 <https://github.com/pantsbuild/pants/pull/8823>`_
+
+* Provide input_files Digest to InteractiveRunner (#8855)
+  `PR #8855 <https://github.com/pantsbuild/pants/pull/8855>`_
+
+* Support enums when displaying option help choices (#8853)
+  `PR #8853 <https://github.com/pantsbuild/pants/pull/8853>`_
+
+* Hotfix MyPy not working on macOS due to type alias (#8887)
+  `PR #8887 <https://github.com/pantsbuild/pants/pull/8887>`_
+
+* Use raw strings to fix regex deprecation warnings (#8880)
```

Comment:
Nit: More a refactoring because this only impacts internal codebase. Don't worry about changing unless you end up making other changes